content/en/docs/setup/best-practices/certificates.md

```diff
@@ -38,11 +38,13 @@ etcd also implements mutual TLS to authenticate clients and peers.
 
 ## Where certificates are stored
 
-If you install Kubernetes with kubeadm, certificates are stored in `/etc/kubernetes/pki`. All paths in this documentation are relative to that directory.
+If you install Kubernetes with kubeadm, most certificates are stored in `/etc/kubernetes/pki`. All paths in this documentation are relative to that directory, with the exception of user account certificates (which are in `/etc/kubernetes`).
+The following diagram:
```

Comment:
```suggestion
```
this seems a left over

Reply:
IMO, ideally diagrams should not highlight the `/etc/kubernetes/pki` tree structure, because it is a kubeadm artifact. other deployers use their own directory structures and this document is not about kubeadm only.

Reply:
Thanks for catching the extra text. I'll only note that the user account certificates in relation to kubeadm and remove them from the diagrams.